packages/apollos-church-api/src/data/features/data-source.js

```diff
@@ -19,4 +21,22 @@ export default class Features extends baseFeatures.dataSource {
       __typename: 'HeaderFeature',
     };
   }
+
+  // TODO come up with a better way to hide features per block
+  // currently this is only showing features if you're on staff
+  async getHomeFeedFeatures() {
+    const { Auth } = this.context.dataSources;
+    const person = await Auth.getCurrentPerson();
+    const staff = await this.request('GroupMembers')
+      .andFilter(`GroupId eq 3`)
+      .get();
```

Comment:
I think we'll need to add a filter to make sure that the group member is active. I'm still seeing the staff block even though I set myself to inactive in the group. I think we can do something similar to what we do for groups in core. 
https://github.com/ApollosProject/apollos-prototype/blob/1fdc94a22ec5884b7549e6df079c86caec512aad/packages/apollos-data-connector-rock/src/groups/data-source.js#L20